packages/trpc/server/routers/loggedInViewer/_router.tsx

```diff
@@ -168,6 +170,21 @@ export const loggedInViewerRouter = router({
       return UNSTABLE_HANDLER_CACHE.appCredentialsByType({ ctx, input });
     }),
 
+  appCredentialById: authedProcedure.input(ZAppCredentialByIdInputSchema).query(async ({ ctx, input }) => {
```

Comment:
Needed to fill Credential Form. Only thing left is returning parital/cloaked ApiKey to avoid exposing it, but projectID and endpoint are definitely something the user has to see.